oaknational/Oak-Web-Application
# Default `null` to empty string (`""`) in curriculum visualiser

## Description
Fix for unselected radio button when first displayed. 

## Issue(s)

Fixes `CUR-1032`

## How to test

1. Go to https://deploy-preview-2958--oak-web-application.netlify.thenational.academy
2. Navigate to curriculum visualiser
3. Check threads are defaulting to "none"
4. Check for regressions
